lambda_utils/Makefile

```diff
@@ -0,0 +1,29 @@
+export ROOT = $(shell git rev-parse --show-toplevel)
+export LAMBDA_UTILS = $(ROOT)/lambda_utils
+export INFRA_BUCKET = platform-infra
+
+ifneq ($(ROOT), $(shell pwd))
+	include $(ROOT)/shared.Makefile
+endif
+
+
+lambda_utils-test: $(ROOT)/.docker/python3.6_ci
+	$(ROOT)/builds/docker_run.py --aws -- \
+		--volume $(LAMBDA_UTILS):/data \
+		--env OP=test \
+		--env FIND_MATCH_PATHS="/data" \
+		python3.6_ci:latest
+
+lambda_utils-build:
+	docker run \
+		--volume $(LAMBDA_UTILS):/src \
+		--workdir /src \
+		python:3-alpine python setup.py sdist
+
+lambda_utils-publish: lambda_utils-build
```

Comment:
This should be `deploy`

Reply:
Mmm. It’s publishing a new version of something to a package repository, but not actually deploying it anywhere in our stack. I prefer "publish", a la ECR publishing.

Reply:
It should be publish, but our build system relies on this being "deploy".

Reply:
No, strike that run_task.py says otherwise.